tests/snappi_tests/intf_utils/intf_accept_metrics.py

```diff
@@ -0,0 +1,89 @@
+# This file defines the interfaces that snappi tests accept external metrics.
+
+import logging
+import json
+import datetime
+import time
+
+from typing import List, Dict, Union
+from intf_report_metrics import MetricReporterFactory OtelMetricReporter
+
+class Metric:
+    def __init__(self,
+                 name: str,
+                 description: str,
+                 unit: str,
+                 timestamp: int,
+                 device_id: str,
+                 component_id: str,
+                 reporter: MetricReporterFactory,
+                 metadata: Dict[str, str]):
+        """
+        Args:
+            name (str): metric name (e.g., psu power, sensor temperature, port stats, etc.)
+            description (str): brief description of the metric
+            unit (str): metric unit (e.g., seconds, bytes)
+            timestamp (int): UNIX Epoch time in nanoseconds when the metric is collected
```

Comment:
if the timestamp is for logging the collection time, the reporter already has it and can be removed